.gitlab-ci.yml

```diff
@@ -38,7 +38,11 @@ test-server_api:
       done
 
 test-simple:
-    script: stack --no-terminal test --ta "-p simple"
+    script:
+    - stack --no-terminal test --ta "-p simple"
+    - apt-get update
+    - apt-get install --assume-yes libgoogle-perftools-dev
```

Comment:
Any objections if I add this to Dockerfile instead?

Reply:
No objection. We can also leave it out entirely if you think it doesn't help much and isn't worth the effort.

Reply:
Pushed to you branch. Hope it's ok.